grails-app/assets/javascripts/streama/services/api-service.js

```diff
@@ -162,7 +162,22 @@ angular.module('streama').factory('apiService', function ($http, $rootScope, con
       }
     },
 
-		episode: {
+    subtitle: {
+      setDefault: function(subtitleId, videoId) {
+        return $http.get('subtitles/setDefault.json', {params: {id: subtitleId, videoId: videoId}});
+      },
+      getOpensubtitles: function (video, videoTitle, subtitlelanguage, isSearchByHash) {
+        return $http.get('subtitles/get.json', {params: {episode: video.episode_number, query: videoTitle, season: video.season_number, subLanguageId: subtitlelanguage, videoId: video.id, searchByHash: isSearchByHash}});
+      },
+      uploadOpensubtitles: function (opensubtitle, videoId) {
+        return $http.get('subtitles/download.json', {params: {subFileName: opensubtitle.subFileName, subDownloadLink: opensubtitle.subDownloadLink, subLang: opensubtitle.languageName, videoId: videoId}});
+      },
+      refreshSubtitles: function (videoId) {
+        return $http.get('subtitles/getVideoSubtitles.json', {params: {videoId: videoId}});
+      },
+    },
```

Comment:
Oh god, our old API structure is so embarrassing 🙄 
But thanks for keeping the style consistent, that way this can be refactored as one at a later date :)